app/javascript/orangelight/google_books_snippets.es6

```diff
@@ -0,0 +1,28 @@
+export default class GoogleBooksSnippets {
+    get has_figgy_viewer() {
+        return $(".document-viewers > .intrinsic-container").length > 0
```

Comment:
I'm realizing this implementation would also return true if it's there's a google books viewer. We're not using it that way but we should either rename it or make it only return true if there's a figgy viewer.